oak_remote_attestation/src/attester.rs

```diff
@@ -77,3 +81,68 @@ impl Attester {
         })
     }
 }
+
+struct _ApplicationPublicKeys {
+    encryption_public_key: Option<Vec<u8>>,
+    signing_public_key: Option<Vec<u8>>,
+}
+
+// TODO(#4074): Rename to `Attester` and replace the existing `Attester`once
+// DICE attestation is implemented.
+/// DICE attester implementation.
+/// <https://www.rfc-editor.org/rfc/rfc9334.html#name-attester>
+struct _DiceAttester {
```

Comment:
Having thought about this a bit more, this struct is an exact copy of DiceData, apart form the fact that the fields are not options. Why not just as some implementations on top of DiceData?